redis_hashring/__init__.py

```diff
@@ -212,32 +218,24 @@ def debug_print(self):
             delay = max(replica[3] for replica in v)
             expired = any(replica[4] for replica in v)
             count = len(v)
+            expired_str = '(EXPIRED)' if expired else ''
             print(
-                '{size:5.2f}% {count:8} {delay:6}s {hostname:20} {pid:5}{extra}'.format(
-                    start=start,
-                    count=count,
-                    hostname=hostname,
-                    pid=pid,
-                    delay=delay,
-                    size=size,
-                    extra=' (EXPIRED)' if expired else '',
-                )
+                f'{size:5.2f}% {count:8} {delay:6}s {hostname:20} {pid:5}'
+                f' {expired_str}'
```

Comment:
I guess we can move this space to above where we set `expired_str`.

Reply:
I prefer the code as it is, it seems cleaner to me this way. Let me know if you have a strong preference otherwise, though.